### DISCUSSION TITLE
Meta_Hours_11

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#3344] [Meta-Hours]
![replicate-prediction-742eyxnrwfes3ixlgwhfrw5hfe](https://user-images.githubusercontent.com/5887929/177547723-af54093a-d99d-43a0-a951-6b017ed77e9e.jpg)

Add to your Calendar 🥺
https://discord.com/events/840514076538830888/994277212792574065

# Basilisk
We are finishing the v0 implementation of Basilisk, which brings tons of new features and trustless NFTs for the Dotsama NFT landscape. You can read about it in previous Meta_Hours #3030  
- #3368 
- Currently, testing is in progress, and we are looking to get **at least 100 active wallets on Basilisk testnet** 
- Do you want to help us with testing? Feel free signup in form https://form.kodadot.xyz/basilisk-nft
- Check amount of active wallets at https://beta.kodadot.xyz/bsx/stats/
<img width="790" alt="image" src="https://user-images.githubusercontent.com/5887929/179727700-9035ecdf-b65f-4b83-b27b-ed35968e2482.png">

# Moonsama - work in progress
- With Moonsama, we will be soft-landing EVM into KodaDot. 
- We will be the first cross-chain EVM NFT explorer on Polkadot / Kusama and Dotsama ecosystem, then we can start supporting EVM collections like GLMRApes and Damned Pirates Society &c
- If you are the owner/creator of a popular EVM collection, feel free to reach us out, best Clairee or Luu
- **You can give like to our submission** to Polkadot North America bounty https://devpost.com/software/kodadot-moonsama
- #3340 

<img width="1034" alt="image" src="https://user-images.githubusercontent.com/5887929/179351469-1965660e-3d5e-44c7-bf7d-e0d1a3d82393.png">

# Uniquery 
- With upcoming more complicated queries, we would like to enable others to leverage our work towards indexers (SubSquid, SubQuery) and try to build various applications with us.  
- https://github.com/kodadot/packages/tree/main/uniquery

# Generative Art

- With the upcoming minimal Dall-e mega implementation, we can think of users to have enabled minting various works faster with prompts 
- We will slowly extending it with various on-chain experiments 
- https://github.com/kodadot/nft-gallery/discussions/3335

# QA-Guild

As mentioned here
- https://github.com/kodadot/nft-gallery/discussions/3399
- We will accelerate our testing cycle and invite existing team members and external contributors to help us with testing PRs. The code-review guild has had some hiccups, so it's better to go with a specialized approach. 
- **If you would participate in our QA guild**, hit me or someone from the team, and we can add you. 
- Currently, 4 members would like to help us with this, and the goal is to have at least ten members to split load well as with each upcoming integration, **we are getting complexity harder**, so it's easier to miss some bugs. As usual, it's part of our releasing process, which could help us release less broken code to the wild. As KodaDot has more users coming, we want to release battle-tested code only. 

## Developers Grant program

- As we have tons of ideas we could not fulfil ourselves and see their potential, we would like slowly open our grants program for incoming developers who would like to contribute to the KodaDot ecosystem in some way. I've briefly drafted RFPs and if you have something you would like to see, let us know; we are happy to review, give it a shot and help you design deliverables to see things working for the bigger audience around KodaDot & MetaPrime
- Feel free bring your idea, we are quite flexible in terms of API and coding creativity hacks
- https://github.com/kodadot/nft-gallery/discussions/3448

## KodaDot team opening web3 cowork in Slovenia

- We've thought that remote works give much flexibility, but we would need some place on planet Earth to meet somewhere physically. Hence, most team members are based in Europe; we've found a potential well-being place in Bled, Slovenia, in the beautiful nature of the Julian Alps, close to national park Triglav. 

- We are proud to open our web3 co-work called [SubWork](https://subwork.xy), where we are inviting all Substrate builders to hang out and nurture meaningful relationships in amazing nature. Of course, any web3 folks are more than welcome to show up as we've noticed that few of them are living in a close radius :)

#### Why? 
We've found out that conferences are a great place to meet with folks, but as it's packed in a short time frame, it feels like taking 10+ Zoom calls in one day, so one doesn't carry that many deep relationships from that day. 
That's recommended stay at SubWork is for 2-3 weeks, you can off course stay even longer. 
It offers you a balanced time to visit national Triglav park, great hikes in the radius, swimming in alpine lake Bohinj, hiking beautiful mountains like Mangart, making your first 100km on a road bike, or taking mountain bike to ride downhill trails. All of that around with tons of camping sites, so if you are into van life, you'll love it here :)

Some people suggest we could do an opening party and invite more folks (tg channel crossed 40+ members). Well, yes, but if someone would like to coordinate this, it would be nice!:)

**Comment by AuspiciousMuna** [comment]
Amazing developments from the team members. Keep it up :)

**Comment by DeluneClaire** [comment]
Claire's minute:
- Program
- Marketing needs
- Reddit event
- Basilisk campaign
- Artists campaign
- Playtime

**Comment by damskyftw** [comment]
Hourly chart:

- Ambassadors direction change, we are looking for you
- Various collaborations, we want to see you
- Collection ranking 
- Introducing Artist support program
- Discord suggestions, ad #3246

**Comment by petersopko** [comment]
quick minutes

- QA guild up and running
- Automated E2E incoming
- Changes to UI
- Basilisk critical issues
- Snek testers influx

**Comment by vikiival** [comment]
**Viki's Meta Minutes**
-

### DEV in numbers:
`is:pr is:merged created:>=2022-05-19`

> 121 PRs merged

### Internal:

Last weeks has been pretty rough as we found out that many of newly submitted issues are coming from a single source - our internal implementation of the Polkadot API.

As it has been stable since April 2020 - with the newly added multi-chain features (BSX, Moonsama) - we found out that the API is not so flexible.

We spent weeks with design and therefore implementation.


What's the output? 

Actually pretty cool. We were able to tweak the Polkadot API so it better suites our needs:

- multiple, non-blocking connections
- auto disconnect
- auto reconnect
- HTTP implementation for bots/tools & so
- built in api utils - what's the block & so

In the term of numbers there are already **100+ commits** in packages and main repo.


Wen? 

I already made a release for npm
And PR for the nft-gallery should be available after metahours

### Click:

as we mentioned in the metahours 10. We have Implemented click

**Comment by JustLuuuu** [comment]
Luuu:
- Artists campaign update
- Reddit news
- Discord (+wagmedia)
- QA guild
